lib/plugins/basalprofile.js

```diff
@@ -6,8 +6,18 @@ function init() {
     name: 'basal'
     , label: 'Basal Profile'
     , pluginType: 'pill-minor'
+    , additionalHtml: '<input type="checkbox" id="basals-switch" style="margin:0px 0px 0px 0px;vertical-align: bottom;">'
   };
 
+  basal.htmlInitCode = function htmlInitCode () {
+    $('#basals-switch').change(function switchChart (event) {
+      window.Nightscout.client.chart.basals.attr('display', $('#basals-switch').is(':checked') ? '' : 'none');
+      if (event) {
+        event.preventDefault();
```

Comment:
maybe a `event.stopPropagation()` here would stop the tooltip from showing on toggle?